operations/helm/charts/mimir-distributed/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-version: 5.4.0
+version: 5.4.1
```

Comment:
normally we don't do patch releases for these fixes. There are [weekly versions released](https://artifacthub.io/packages/helm/grafana/mimir-distributed/5.4.0-weekly.298) every Monday (usually on time) which contain the changes on main. Will these weekly releases work for you?

Reply:
> Normally contributors need not bump the version.

My mistake, totally missed that. Weekly is definitely fine, I'll revert that change shortly.